project/src/components/sort/sort.tsx

```diff
@@ -1,18 +1,51 @@
-function Sort(): JSX.Element {
+import {SORT_TYPE} from '../../const';
+import classNames from 'classnames';
+import {useState} from 'react';
+
+type SortProps = {
+  sortTypeCheck: string,
+  onSortClick?: (sortType: string) => void;
+}
+
+function Sort({sortTypeCheck, onSortClick}: SortProps): JSX.Element {
+  const [show, setState] = useState(false);
+
+  const handleSortListOpen = () => {
+    if (!show) {
+      setState(true);
+    } else {
+      setState(false);
+    }
+  };
+
+  const sortListClass = classNames(
+    'places__options',
+    'places__options--custom',
+    {'places__options--opened': show},
+  );
+
   return (
-    <form className="places__sorting" action="#" method="get">
+    <form className="places__sorting" action="#" method="get" onClick={handleSortListOpen}>
```

Comment:
Лучше действие использовать в нейминге
```suggestion
    <form className="places__sorting" action="#" method="get" onClick={handleSortClick}>
```